### ISSUE TITLE
Unable to access 2 links when swiping in VO

### ISSUE THREAD

**Issue by paulettefrancis** [cds-snc/covid-shield-mobile#207]
Swipe in VO.

Should be able to access "francais" and "next" at the bottom of the pages. 

On the set of Onboarding pages, starting with "What COVID Alert does", when swiping in VO, "francais" and "next" are not accessible as it goes to the next page in the set automatically when swiping after the last item.

iPhone XR 13.5.1

**Comment by JuliannaR** [comment]
@paulettefrancis the francais link has been removed.

**Comment by smcmurtry** [comment]
This was fixed when we moved to the page navigation in #398
